src/test/munit/arguments-transformation-test-suite.xml

```diff
@@ -0,0 +1,344 @@
+                </java:args>
+            </java:invoke-static>
+        </munit:execution>
+        <munit:validation>
+            <munit-tools:assert-that doc:name="Assert"
+                                     expression="#[payload]" is="#[MunitTools::equalTo('4')]"/>
+        </munit:validation>
+    </munit:test>
+
+    <munit:test name="invokeStaticWithListOfMapsAndStringsTransformed">
+        <munit:execution>
+            <java:invoke-static
+                    class="org.mule.extensions.internal.model.AnotherExecutableElement"
+                    method="getFromMapInsideListAndSumFour(List,int,String)">
+                <java:args>
+                    #[{
+                    'listOfMaps': [{
+                    'hello' : "33",
+                    'cat' : "44",
+                    'dog' : "22"
+                    },{
+                    'fish' : "11",
+                    'water' : "24",
+                    'mac' : "65"
+                    }],
+                    'index' : "1",
+                    'key' : "water"
+                    }]
+                </java:args>
+            </java:invoke-static>
+        </munit:execution>
+        <munit:validation>
+            <munit-tools:assert-that doc:name="Assert"
+                                     expression="#[payload]" is="#[MunitTools::equalTo(28)]"/>
+        </munit:validation>
+    </munit:test>
+
+    <munit:test name="invokeStaticWithNullParameter" description="MUnit Test" ignore="true">
+        <munit:behavior>
+            <set-variable variableName="nullValue"
+                          value="#[java!org::mule::extensions::internal::model::ExecutableElement::getNull()]"/>
+        </munit:behavior>
+        <munit:execution>
+            <java:invoke-static
+                    class="org.mule.extensions.internal.model.AnotherExecutableElement"
+                    method="getSameWord(String)">
+                <java:args>
+                    #[{ 'word' : payload} ]
+                </java:args>
+            </java:invoke-static>
+        </munit:execution>
+        <munit:validation>
+            <munit-tools:assert-that doc:name="Assert"
+                                     expression="#[payload]" is="#[MunitTools::nullValue()]"/>
+        </munit:validation>
+    </munit:test>
+
+    <munit:test name="invokeStaticWitNullParameterForPrimitiveValue" expectedErrorType="JAVA:ARGUMENTS_MISMATCH"
+                description="MUnit Test" ignore="true">
+        <munit:behavior>
+            <set-variable variableName="nullValue"
+                          value="#[java!org::mule::extensions::internal::model::ExecutableElement::getNull()]"/>
+        </munit:behavior>
+        <munit:execution>
+            <java:invoke-static
+                    class="org.mule.extensions.internal.model.AnotherExecutableElement"
+                    method="getSameNumber(int)">
+                <java:args>
+                    #[{ 'number' : vars.nullValue } ]
```

Comment:
removing this and replacing it with vars.missingValue, should produce the same effect

Reply:
I removed the var declaration, but leaving the name as `nullValue` seems more descriptive